pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/assignment/segment/OfflineAllServersSegmentAssignmentStrategy.java

```diff
@@ -51,23 +50,24 @@
  *   </li>
  * </ul>
  */
-public class OfflineDimTableSegmentAssignment implements SegmentAssignment {
+public class OfflineAllServersSegmentAssignmentStrategy implements SegmentAssignmentStrategy {
```

Comment:
Let's actually take it back as `DimTableSegmentAssignmentStrategy`

Reply:
done